packages/x-components/src/x-modules/tagging/wiring.ts

```diff
@@ -93,6 +93,16 @@ export const trackResultClickedWire = filter(
   ({ eventPayload: { tagging } }) => !!tagging?.click
 );
 
+/**
+ * Tracks the tagging of the add to cart.
```

Comment:
`track` and `tag` are synonyms in this context. My proposal

```suggestion
 * Performs a track of a result added to the cart.
```